rep-2000.rst

```diff
@@ -750,11 +865,15 @@ References and Footnotes
 .. [5] Dashing ROS 2 Repos File
    (https://github.com/ros2/ros2/blob/dashing/ros2.repos)
 .. [6] Eloquent ROS 2 Repos File
-   (https://github.com/ros2/ros2/blob/master/ros2.repos)
+   (https://github.com/ros2/ros2/blob/eloquent/ros2.repos)
 .. [7] Connext DDS 6.0.0 Support
    (https://github.com/ros2/rmw_connext/issues/375)
 .. [8] REP 2002
    (http://www.ros.org/reps/rep-2002.html)
+.. [9] Foxy ROS 2 Repos File
+   (https://github.com/ros2/ros2/blob/foxy/ros2.repos)
+.. [10] Galactic ROS 2 Repos File
+   (https://github.com/ros2/ros2/blob/master/ros2.repos)
```

Comment:
Note that this link should be updated after we branch ([circa April](https://index.ros.org/doc/ros2/Releases/Release-Galactic-Geochelone/#timeline-before-the-release)).